ground/src/main/java/com/google/android/ground/persistence/local/room/RoomLocalDataStore.kt

```diff
@@ -508,7 +506,7 @@ class RoomLocalDataStore @Inject internal constructor() : LocalDataStore {
    * Applies mutation to submission in database or creates a new one.
    *
    * @return A Completable that emits an error if mutation type is "UPDATE" but entity does not
-   * exist, or if type is "CREATE" and entity already exists.
+   *   exist, or if type is "CREATE" and entity already exists.
```

Comment:
This extra space is causing `ktfmtCheck` to fail. `ktfmtFormat` removes it while Android Studio's code formatter adds it. For now, let's just fix this manually to unblock this PR and tackle this issue separately.

Reply:
Funny, I thought I tried removing it and ktfmt still failed. Part of the problem is there's no line number or other hints reported by ktfmt, so hard to know where the actually error occurs. Trying this now.